tests/replication_upgrade.erl

```diff
@@ -4,6 +4,13 @@
 -include_lib("eunit/include/eunit.hrl").
 
 confirm() ->
+
+    OldDelay = rt_config:get(rt_retry_delay, 1000),
+    lager:info("OldDelay is ~p", [OldDelay]),
+    rt_config:set(rt_retry_delay, 5000),
```

Comment:
I'm a bit concerned about hard-coding a timeout.  How about something a bit more flexible, but works just the same: `OldDelay * 5`